site2/docs/security-token-client.md

```diff
@@ -9,20 +9,18 @@ sidebar_label: Client Authentication using tokens
 Pulsar supports authenticating clients using security tokens that are based on
 [JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
 
-Tokens are used to identify a Pulsar client and associate with some "principal" (or "role") which
-will be then granted permissions to do some actions (eg: publish or consume from a topic).
+You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that
+is permitted to do some actions (eg: publish or consume from a topic) then.
```

Comment:
1. do not use abbreviation in technical writing, not use "eg, etc. ...", you can use "for example", "such as".
2. complete information: publish messages to a topic, consume messages from a topic.
3. delete "then" at the end.